lua/lspconfig/server_configurations/clangd.lua

```diff
@@ -2,10 +2,15 @@ local util = require 'lspconfig.util'
 
 -- https://clangd.llvm.org/extensions.html#switch-between-sourceheader
 local function switch_source_header(bufnr)
+  local clangd_client = nil
```

Comment:
you probably want to extract that to a `util` function `util.get_buf_client(bufnr, servername)` or some such.